### PR TITLE
drop v1.10 support

### PR DESCRIPTION
**Pull request by aanm**
v1.10 has reached EOL so we are removing any references of it in the code tree.

### CODE REVIEW

File: Documentation/contributing/testing/ci.rst [hunk 1]
```diff
@@ -29,11 +29,11 @@ them all at once:
 +==================+==========================+
 | master           | /test                    |
 +------------------+--------------------------+
-| v1.12            | /test-backport-1.12      |
+| v1.13            | /test-backport-1.12      |
```
Comment (tklauser) [inline Documentation/contributing/testing/ci.rst]:
```suggestion
| v1.13            | /test-backport-1.13      |
```

File: Documentation/contributing/testing/ci.rst [hunk 2]
```diff
@@ -29,11 +29,11 @@ them all at once:
...
 +------------------+--------------------------+
-| v1.11            | /test-backport-1.11      |
+| v1.12            | /test-backport-1.13      |
```
Comment (tklauser) [inline Documentation/contributing/testing/ci.rst]:
```suggestion
| v1.12            | /test-backport-1.12      |
```

**Review by tklauser — APPROVED** [review]
LGTM apart from a mixup in the `/test-backport-1.1[23]` trigger phrases.

**Review by qmonnet — APPROVED** [review]

**Review by joestringer — APPROVED** [review]
LGTM. I guess we also need to update the algolia script.

**Review by nbusseneau — APPROVED** [review]

**Comment by nbusseneau** [comment]
Can I delete this now? Please please pretty please? https://jenkins.cilium.io/view/Cilium-v1.10/

**Review by nbusseneau — APPROVED** [review]

**Pull request merged**